MovieSharing/APIManager.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  VideoRequest.swift
+//  MovieSharing
+//
+//  Created by Mariam Korashi on 29.09.21.
+//
+
+import Foundation
+
+class APIManager {
+    
+    // MARK: - Properties
+    
+    static let sharedAPIManager = APIManager()
```

Comment:
- [ ] I suggest renaming to just something like `shared` as you otherwise call this by using `APIManager.sharedAPIManager` which creates a bit of a naming redundancy